csuntechlab/iSTART
# STR-213 - FE: Clicking continue and Back go to the Top of the Content.

# Description
When going through the modules the viewport should go to the top of content on every slide. Instead of ending up on the previous slide y-axis position.
  
# Testing
1. Select a module.
2. You might want to change your browser viewport to mobile.
3. Click on Continue or Back
4. The viewport should reset to the top of the content.
